.travis.yml

```diff
@@ -34,7 +34,7 @@ matrix:
         - CMD="mvn install --projects jmh-tests,performance-tests --also-make --activate-profiles all -DskipTests=true --batch-mode --show-version --errors"
 
 script:
-- if [[ "$TRAVIS_EVENT_TYPE" == "cron" && DESC="unit tests" ]];
+  if [[ "$TRAVIS_EVENT_TYPE" == "cron" && "$DESC" == "unit tests" ]];
   then mvn install --batch-mode --show-version --errors sonar:sonar -Dsonar.host.url=https://sonarqube.com -Dsonar.login=${SONARQUBE_TOKEN} -Dsonar.organization=eclipse-collections ;
```

Comment:
This should really be implemented at the top of the file and should build up a "sonar command." Then just change the unit tests command in the matrix above.

```
     - jdk: oraclejdk8
       env:
         - DESC="unit tests"
         - CMD="mvn install --batch-mode --show-version --errors $SONAR_COMMAND"
```

Where $SONAR_COMMAND is allowed to be empty.